src/components/table/columns.ts

```diff
@@ -34,6 +38,27 @@ export class ColumnDefinitionFactory {
     }
 }
 
+export const formatTitle = (column: Column) => (): string | HTMLElement => {
+    const element = document.createElement('div');
+    element.innerText = column.title;
+
+    const icon = document.createElement('limel-icon');
+    icon.setAttribute('name', column.menuIcon);
+    icon.setAttribute('size', 'x-small');
```

Comment:
I am not sure if we set the icon size here

Reply:
How would the consumer set the size with the implementation right now? :slightly_smiling_face: It would be great with an example!
(On a side note I'm not sure the consumer should be allowed to change the size, but that's another discussion)

Reply:
I was thinking if we can add a property called `headerIconSIze` to the column interface

Reply:
No we should totally set the size ourselves! It's 100% ours.